node/resolvers/search/index.ts

```diff
@@ -418,23 +455,50 @@ export const queries = {
 
   products: async (_: any, args: SearchArgs, ctx: Context) => {
     const {
-      clients: { search },
+      clients: { biggySearch },
+      vtex: { segment }
     } = ctx
-    const queryTerm = args.query
-    if (queryTerm == null || test(/[?&[\]=]/, queryTerm)) {
+    const {query, to, from, orderBy, simulationBehavior} = args;
+
+    if (query == null || test(/[?&[\]=]/, query)) {
       throw new UserInputError(
-        `The query term contains invalid characters. query=${queryTerm}`
+        `The query term contains invalid characters. query=${query}`
       )
     }
 
-    if (args.to && args.to > 2500) {
+    if (to && to > 2500) {
       throw new UserInputError(
         `The maximum value allowed for the 'to' argument is 2500`
       )
     }
-    const products = await search.products(args)
-    searchFirstElements(products, args.from, ctx.clients.search)
-    return products
+
+    const selectedFacets: SelectedFacet[] = buildSelectedFacets(args)
+
+    const biggyArgs: SearchResultArgs = {
+      fullText: query,
+      attributePath: buildAttributePath(selectedFacets),
+      tradePolicy: segment && segment.channel,
+      query: query,
+      operator: "and"
+    };
+
+    if(orderBy){
+      biggyArgs["sort"] = convertOrderBy(orderBy);
+    }
```

Comment:
```suggestion
      sort: convertOrderBy(orderBy)
    }
```

this function `convertOrderBy` has already a check for `orderBy`

Reply:
The function `convertOrderBy(orderBy)` can receive `string | undefined` but the `orderBy` can be `string | null` so it doesn't work without the check :/

Reply:
in this case you can just add the type `null` to the `convertOrderBy` param, so you don't need to check this in two places.